### ISSUE TITLE
Firefox always uses POST

### ISSUE THREAD

**Issue by jeroenev** [searx/searx#1918]
When adding the search engine to FireFox, it always seems to use POST instead of GET.
Even though I set the preferences on the site to use GET.
Is there any way to fix this so that firefox uses get-parameters from the URL/Search bar?

**Comment by guidot** [comment]
I cannot reproduce that on Firefox 74.0.1 (Linux).
I tested it on self hosted instance. After setting it to GET and submitting a search for "fax" the result page shows the term in the URL: searx.example.org/?q=fax

**Comment by jeroenev** [comment]
This is how it looks for me: https://intuxikated.stackstorage.com/s/055tojPxxuacTB4
When I use the regular site's search bar, it does search using get parameters.
However, If I try to add the search engine to firefox, and then search using the Firefox search bar, It does not use get-parameters.

**Comment by MageJohn** [comment]
I'm experiencing this issue as well. The instance I'm using is searx.canox.net, and my current version of Firefox is 75.0 on Manjaro Linux. I've tried removing searx, setting the preferences to use GET, and then readding searx using the + icon in the search bar, and the problem persists. I've also looked at the content of the opensearch.xml file, which seems to specify GET: 

```xml
<?xml version="1.0" encoding="UTF-8"?>
<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
  <ShortName>searx.canox.net</ShortName>
  <Description>a privacy-respecting, hackable metasearch engine</Description>
  <InputEncoding>UTF-8</InputEncoding>
  <Image>https://searx.canox.net/static/themes/oscar/img/favicon.png</Image>
  <LongName>searx metasearch</LongName>
    <Url type="text/html" method="get" template="https://searx.canox.netsearch?q={searchTerms}"/>
    <Url type="application/x-suggestions+json" method="get" template="https://searx.canox.netautocompleter">
        <Param name="format" value="x-suggestions"/>
        <Param name="q" value="{searchTerms}"/>
    </Url>
</OpenSearchDescription>
```

**Comment by michaelschufi** [comment]
This has been solved afaik by #2247 (also see #2245).
